instrumentation/README.md

```diff
@@ -1,181 +1,129 @@
--Djava.io.tmpdir=/usr/local/apache-tomcat/8.5.4/temp \
-org.apache.catalina.startup.Bootstrap start
-```
-
-produces a service name of:
-
-```
-org-apache-catalina-startup-bootstrap
-```
-
-## Disabling
-
-Short of uninstalling the package, there are a few safe ways of disabling auto instrumentation
-
-* Set `DISABLE_SPLUNK_AUTOINSTRUMENTATION` to any non-empty value other than false, FALSE, or 0
-* Set `JAVA_TOOL_OPTIONS` to some value that you want the JVM to pick up
-* Delete or move the config `instrumentation.conf` file
-
-## Files
-
-* /usr/lib/splunk-instrumentation/libsplunk.so
-* /usr/lib/splunk-instrumentation/instrumentation.conf
-* /usr/lib/splunk-instrumentation/splunk-otel-javaagent.jar
+# Splunk OpenTelemetry Zero Configuration Auto Instrumentation for Linux
+
+The **Splunk OpenTelemetry Zero Configuration Auto Instrumentation for Linux** Debian/RPM package
+(`splunk-otel-auto-instrumentation`) installs Splunk OpenTelemetry Auto Instrumentation agents, the `libsplunk.so`
+shared object library, and default/sample configuration files to automatically instrument applications and services to
+capture and report distributed traces and metrics to the [Splunk OpenTelemetry Collector](
+https://docs.splunk.com/Observability/gdi/opentelemetry/opentelemetry.html), and then on to [Splunk APM](
+https://docs.splunk.com/Observability/apm/intro-to-apm.html).
+
+The `splunk-otel-auto-instrumentation` deb/rpm package installs and supports configuration of the following Auto
+Instrumentation agents:
+
+- [Java](https://docs.splunk.com/Observability/gdi/get-data-in/application/java/get-started.html)
+- [Node.js](https://docs.splunk.com/Observability/en/gdi/get-data-in/application/nodejs/get-started.html)
+
+For other languages or if the `splunk-otel-auto-instrumentation` deb/rpm package is not applicable for the target host
+or applications/services, see [Instrument back-end applications to send spans to Splunk APM](
+https://docs.splunk.com/Observability/en/gdi/get-data-in/application/application.html).
+
+## Prerequisites/Requirements
+
+- Check agent compatibility and requirements:
+  - [Java](https://docs.splunk.com/Observability/gdi/get-data-in/application/java/java-otel-requirements.html)
+  - [Node.js](https://docs.splunk.com/Observability/en/gdi/get-data-in/application/nodejs/nodejs-otel-requirements.html)
+- [Install and configure](https://docs.splunk.com/Observability/gdi/opentelemetry/install-linux.html) the Splunk
+  OpenTelemetry Collector.
+- Debian or RPM based Linux distribution (amd64/x86_64 or arm64/aarch64).
+
+## Installation
+
+### Manual
+
+1. [Install and configure](https://docs.splunk.com/Observability/gdi/opentelemetry/install-linux.html) the Splunk
+   OpenTelemetry Collector
+2. [Install](../docs/getting-started/linux-manual.md#auto-instrumentation-debianrpm-packages) the
+   `splunk-otel-auto-instrumentation` deb/rpm package
+3. If Auto Instrumentation for Node.js is required, [install](
```

Comment:
Couple of questions:
Correct me if I am wrong, for our installers both steps 2 and 3 will be abstracted for customers right?
Is the reason for not including the node.js binary as part of the splunk-otel-auto-instrumentation that contains the java agent because it requires a separate install with npm?

Reply:
- The `splunk-otel-auto-instrumentation` deb/rpm will include the latest `splunk-otel-js` tgz package from https://github.com/signalfx/splunk-otel-js/releases.
- After installing the `splunk-otel-auto-instrumentation` deb/rpm, the `splunk-otel-js` tgz package is then required to be separately installed with `npm`. The plan is to update the installers to try to automate these steps to enable Node.js.